src/components/CardListItem.js

```diff
@@ -21,20 +30,59 @@ export const CardListItem = memo(({ navigation, horizontal, item }) => {
       disabled={!navigation}
     >
       <Card containerStyle={styles.container}>
-        <View style={stylesWithProps({ horizontal }).contentContainer}>
-          {!!picture && !!picture.url && (
-            <Image
-              source={{ uri: picture.url }}
-              style={stylesWithProps({ horizontal }).image}
-              containerStyle={styles.imageContainer}
-              borderRadius={5}
-            />
-          )}
-          {!!subtitle && <RegularText small>{subtitle}</RegularText>}
-          {!!title && (
-            <BoldText>
-              {horizontal ? (title.length > 60 ? title.substring(0, 60) + '...' : title) : title}
-            </BoldText>
+        <View
+          style={[
+            stylesWithProps({ horizontal }).contentContainer,
+            !!contentContainerStyle && contentContainerStyle
+          ]}
+        >
+          {contentSequence?.length ? (
+            // eslint-disable-next-line complexity
+            contentSequence.map((item, index) => (
+              <Fragment key={index}>
+                {item === 'picture' && !!picture && !!picture.url && (
+                  <Image
+                    source={{ uri: picture.url }}
+                    style={stylesWithProps({ horizontal }).image}
+                    containerStyle={styles.imageContainer}
+                    borderRadius={contentContainerStyle.borderRadius || 5}
+                  />
+                )}
+                {item === 'topTitle' && !!topTitle && <RegularText small>{topTitle}</RegularText>}
+                {item === 'subtitle' && !!subtitle && <RegularText small>{subtitle}</RegularText>}
+                {item === 'title' && !!title && (
+                  <BoldText>
+                    {horizontal
+                      ? title.length > 60
+                        ? title.substring(0, 60) + '...'
+                        : title
+                      : title}
+                  </BoldText>
```

Comment:
please do not duplicate this rendering but find a way to have it in one place and reuse it here and there.

Reply:
I'll think about it. But since this was the fastest solution I could find, I developed the code in this way.

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/958/commits/950aa40681b316b9da28daf52e565d258d024ff6